core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java

```diff
@@ -64,7 +64,7 @@
   /**
    * whether to remove certain headers from the 3rd party invocations
    */
-  @InjectProperty(keys = {"request.${op-any-priority}.clientRequestHeaderFilterEnabled",
+  @InjectProperty(keys = {"request.clientRequestHeaderFilterEnabled.${op-any-priority}",
       "request.clientRequestHeaderFilterEnabled"}, defaultValue = "true")
```

Comment:
`{"request.clientRequestHeaderFilterEnabled.${op-any-priority}","request.clientRequestHeaderFilterEnabled"}`

can be simpler:
`{"request.clientRequestHeaderFilterEnabled${op-priority}"}`

Reply:
fixed